README.md

```diff
@@ -51,7 +51,7 @@ There are two ways of citing mirdata:
 
 If you are using the library for your work, please cite the version you used as indexed at Zenodo:
 
-DOI
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4355859.svg)](https://doi.org/10.5281/zenodo.4355859)
```

Comment:
When I click the link it takes me to a page that says "Error: DOI not found"

Reply:
it says it might not have been activated yet (maybe it's too new!) I'll double check in a little bit

Reply:
I reserved it but didn't upload anything in Zenodo yet, that's why is showing like that. It says it will be online when we make the release